kcbq-connector/src/test/java/com/wepay/kafka/connect/bigquery/integration/BigQueryErrantRecordHandlerIT.java

```diff
@@ -73,6 +74,98 @@ public void close() throws Exception {
     }
   }
 
+  @Test
+  public void testRecordsSentToDlqOnInvalidArgumentAvroStorageApi() throws Exception {
+    final String topic = "test-dlq-feature-avro";
+    final String dlqTopic = "dlq_topic";
+
+    createTopicAndTable(topic);
+    Map<String, String> props = connectorAvroProps(topic, dlqTopic);
+    props.put(BigQuerySinkConfig.USE_STORAGE_WRITE_API_CONFIG, "true");
+    // start a sink connector
+    connect.configureConnector(CONNECTOR_NAME, props);
+
+    // wait for tasks to spin up
+    waitForConnectorToStart(CONNECTOR_NAME, 1);
+
+    // Instantiate the converters we'll use to send records to the connector
+    converter = new AvroConverter();
+    converter.configure(Collections.singletonMap(
+                    SCHEMA_REGISTRY_URL_CONFIG,schemaRegistryUrl
+            ), false
+    );
+
+    List<SchemaAndValue> records = getRecords();
+    schemaRegistry.produceRecords(converter, records, topic);
```

Comment:
Minor: lets add a comment here - where we mention we are generating invalid records which leads to INVALID_ARGUMENT error.

Reply:
Nice suggestion, adding.